openapi/ga4gh-tool-discovery.yaml

```diff
@@ -531,8 +537,7 @@ definitions:
           type: string
       has_checker:
         type: boolean
-        description: >-
-          Whether this tool has a checker tool associated with it
+        description: Whether this tool has a checker tool associated with it
```

Comment:
Hmm, I guess it doesn't always say `OPTIONAL` after all. :man_shrugging: I guess you can ignore these comments, but I'll leave them for consideration.